### ISSUE TITLE
dysmetric vertical saccades

### ISSUE THREAD

**Issue by LOURGHI** [obophenotype/human-phenotype-ontology#2678]
_**Please consider a new term for dysmetric vertical saccades observed in Spinocerebellar ataxia type 37.PMID:23700170.
Superclass:Abnormal conjugate eye movement??**_

**Comment by psergouniotis** [comment]
as per ERN-EYE the hierarchy for HP:0000570 should look like this:

Abnormality of saccadic eye movements
  | Selective loss of horizontal saccades (please add this term)
  | Slow saccadic eye   movements
  | | Slowed horizontal saccades (please rename to "Slow horizontal saccades")
  | Hypometric saccades
  | | Hypometric horizontal saccades
  | Dysmetric saccades
  | Hypermetric saccades
  | | Hypemetric horizontal saccades (please add this term)

the term "dysmetric vertical saccades" should be added as a subclass of "Dysmetric saccades (HP:0000641)". 
also, for completeness please consider adding "dysmetric horizontal saccades as a subclass of "Dysmetric saccades (HP:0000641)".

**Comment by psergouniotis** [comment]
please consider making the above recommended modifications and closing this issues. 
please let me know if something is unclear of if you would like any additional information

**Comment by pnrobinson** [comment]
Also adding terms for dysmetric saccades with definition -
Inaccurate saccades (rapid movement of the eye between fixation points) in the vertical direction.
please reopen if this is not accurate!

**Comment by psergouniotis** [comment]
many thanks. please also consider making the above recommended corrections in the hpo tree ("Abnormality of saccadic eye movements" section). please let me know if you would like any definitions etc

**Comment by pnrobinson** [comment]
I did make these changes -- please note that the latest version of the HPO should be online in the browser sometime tomorrow late afternoon! (https://hpo.jax.org/app/browse/term/HP:0000570). The current version on the browser is unfortunately out of date.
